src/v2/ion/ion-type-handlers/string.js

```diff
@@ -0,0 +1,84 @@
+/* eslint max-depth: [2, 3] */
```

Comment:
why name this file string.js ?

Reply:
ditto.

Reply:
sounds like this file will contains functions to operate on strings or would have some string helper functions.

But what it does is, It handles ION string type to convert into meaningful form input types..

Reply:
> sounds like this file will contains functions to operate on strings or would have some string helper functions.

Could be unless you pay attention to the directory name.
I was debating myself whether shall make file name more wording even though the directory name made the hints.

Reply:
basically what i meant was the output is not a string but an object (uiSchema).. string.js, object.js are too general and sounds lower level implementations. Name suggest that we are overriding functioning of string or object here.. But actually it doesn't do that.

Reply:
I meant the directory name could(expect) give you hints. but anyway I'll try to come up another name.